lms/djangoapps/bulk_email/tasks.py

```diff
@@ -209,10 +209,6 @@ def perform_delegate_email_batches(entry_id, course_id, task_input, action_name)
     def _create_send_email_subtask(to_list, initial_subtask_status):
         """Creates a subtask to send email to a given recipient list."""
         subtask_id = initial_subtask_status.task_id
-        log.info(
```

Comment:
What was the issue actually?

Reply:
These logs were being hit before the queue one, and the inside function is called after the queue. To avoid confusion, I updated the logs, already present inside the actual sub-task that mark the beginning of the task, to contain the information of the time when the task initiation started.